app/assets/stylesheets/style.scss

```diff
@@ -24,6 +24,12 @@ body {
   font-size: 80%;
 }
 
+.course-info span img {
```

Comment:
Merge rule `.course-info span img` with rule on line 23<br>Selector should have depth of applicability no greater than 2, but was 3